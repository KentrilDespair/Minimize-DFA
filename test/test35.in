2
cba
2
2
