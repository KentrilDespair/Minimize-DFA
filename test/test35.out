0,1
abc
0
0
0,a,1
0,b,1
0,c,1
1,a,1
1,b,1
1,c,1
